### ISSUE TITLE
CPU frequency tweak base on cores active.

### ISSUE THREAD

**Issue by Godl1nk** [JamesCJ60/Universal-x86-Tuning-Utility#14]
Suggestion: adding a feature which allows us to set cpu frequency according to number of cores active, smtg like CPU turbo ratio limit on throttlestop

**Comment by JamesCJ60** [comment]
I can add something along the lines of this in the future, but it would be limited to the following:
- Unlocked CPUs/APUs
- Systems that support Windows power plan max frequency setting

**Comment by Godl1nk** [comment]
yes im requesting just that "Systems that support Windows power plan max frequency setting"

i request for a feature like ThrottleStop's clock ratio which is only for intel to be added fro amd in this tool, allowing us to set cpu max frequency based on cores active, which u can limi the frequency with windows power plan, just need to add a listener to check number of cores under load

**Comment by JamesCJ60** [comment]
Hello, as this request related to a much older version of UXTU, I will close this issue for the time being.
